tests/test_scripts.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2020 FireEye, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+import os
+import sys
+import subprocess
+
+import pytest
+from fixtures import CD, get_extractor, get_data_path_by_name
+
+
+def get_script_path(s):
+    return os.path.join(CD, "..", "scripts", s)
+
+
+def get_file_path():
+    return get_extractor(get_data_path_by_name("9324d...")).path
+
+
+def get_rules_path():
+    return os.path.join(CD, "..", "rules")
+
+
+def get_rule_path():
+    return os.path.join(get_rules_path(), "lib", "allocate-memory.yml")
+
+
+@pytest.mark.parametrize(
+    "script,args",
+    [
+        pytest.param(
+            "show-features.py",
+            [get_file_path()],
+        ),
+        pytest.param("bulk-process.py", [get_file_path()]),
```

Comment:
I tried to parametrize fixtures, but didn't succeed quickly so went with this instead.